src/i18n_data.ts

```diff
@@ -50,24 +50,327 @@ export const resources = {
+        withdraw: 'Withdraw request',
+        join: 'Join'
+      },
+      groupDetailsPage: {
+        notFound: 'Group not found!',
+        notFound2: 'No group with given name',
+        notFound2Message: `The group you have searched doesn't exist, or has been deleted`,
+        created: 'Created',
+        role: 'Role',
+        pendingMembers: 'Pending members',
+        members: 'Members',
+        groupJoined: 'You have joined the group!',
+        groupDeleted: 'You have deleted the group!',
+        requestWithdrawn: 'You have withdrawn your request!',
+        leftGroup: 'Group left!',
+        makeKonzi: 'Make a consultation',
+        edit: 'Edit',
+        editGroup: '"Edit group',
+        groupEdited: 'Group successfully edited',
+        delete: 'Delete',
+        deleteGroup: 'Delete group',
+        confirmDelete: 'Are you sure you want to delete the group?',
+        leave: 'Leave',
+        leaveGroup: 'Leave group',
+        confirmLeave: 'Are you sure you want to leave this group?',
+        withdrawReq: 'Withdraw request',
+        join: 'Join',
+        userAccpeted: 'User accepted',
+        userRejected: 'User rejected',
+        userPromoted: 'User promoted',
+        userDemoted: 'User demoted',
+        userKicked: 'User kicked',
+        owner: 'Owner',
+        you: 'You',
+        joined: 'Join',
+        accept: 'Accept',
+        reject: 'Reject',
+        demote: 'Demote',
+        promote: 'Promote',
+        kick: 'Kick',
+        noMembers: 'No members',
+        noPendingMembers: 'No pending members'
+      },
+      userList: {
+        allRating: 'Cumulative rating',
+        ratingsForKonzi: 'Ratings for this consultation',
+        rating: 'Rating: '
+      },
+      userRating: {
+        ratingSuccess: 'Rated successfully!',
+        ratingUpdateSuccess: 'Rating successfully updated!',
+        konziNotFound: 'Consultation not found!',
+        yourRating: 'Your rating:',
+        edit: 'Edit',
+        rateAfterStart: 'You can rate the tutor after the start of the consultation.',
+        rate: 'Rate',
+        ratingOf: 'Rate {{user}}',
+        comment: 'Comment (optional)',
+        rating: 'Rating...',
+        anonymus: 'Rate anonymously',
+        anonymusMessage: 'Admins can still see your name when rating anonymously. Please be respectful with your message',
+        save: 'Save'
+      },
+      requestListPage: {
+        supportSuccess: 'You endorsed this request!', //
+        unsupportSuccess: 'You no longer endorse this request!', //
+        requests: 'Consultation requests',
+        newRequest: 'Request new consultation',
+        requestDesc: `Do you need help in a subject? Create a consultation request, and if responds to it, you will be notified via email.
+        If a request already exists from the subject, you may endorse that request; you will still recieve an email notification.`, //
+        noRequests: 'There are no requests!',
+        unsupport: 'Un-endorse', //
```

Comment:
Undo endorsement ?? mondjuk lehet hosszú lesz tőle a gomb

Reply:
am a support - undo/(remove) support nem jobb?  nem vagyok benne biztos hogy az endorse szót minden külföldi hallgató ismerné

vagy akár lehet ne
take part, vagy join is, meg ennek valami az ellentettje

Reply:
Ezen egy ideig tanakodtam, végül azért írtam az endorse-ot, mert a support az kicsit olyan, hogy valamilyen cselekménnyel támogatja, pl. segíti. Az endorse az kicsit inkább annyi, hogy egyetért, pl. politikában egyetért az ideológiájával meg ilyesmi.

Az un-endorse-al kapcsolatban, ezt csak az fogja látni aki már ráklikkelt az endorse-ra szóval fogja tudni, hogy mit csinál a gomb. Az angolok is kitalálnak random szavakat mái napig szóval szerintem nem gond ha un-endorse van ott 😅